julia/README.md

```diff
@@ -0,0 +1,83 @@
+
+- Linting: 
+    - linting is available via VSCode's Julia plugin or through Juno.
+
+## Recommended libraries
+ 
+- Arrays and vectors (numpy analogon): 
+    - [standard library core](https://docs.julialang.org/en/v1/base/arrays/) contains classes for dense, n-dimensional arrays analogous to numpy-arrays. 
+    - [SparseArrays.jl](https://docs.julialang.org/en/v1/stdlib/SparseArrays/): Part of the standard library. Supports sparse vectors and matrices.
+    - [Tensors.jl](https://github.com/Ferrite-FEM/Tensors.jl) and [TensorOperations.jl](https://github.com/Jutho/TensorOperations.jl): For higher-order-Tensor math.
+
+- Linear Algebra: 
+    - [LinearAlgebra.jl](https://docs.julialang.org/en/v1/stdlib/LinearAlgebra/): Standard library module for linear algebra operations like matric traces, in verse, multiplication and so on. 
+
+- Data analysis and data transformation: 
+    - [DataFrames.jl](https://github.com/JuliaData/DataFrames.jl) and [DataFramesMeta.jl](https://github.com/JuliaData/DataFramesMeta.jl) for tabular data. Julia's analog to [Pandas](https://pandas.pydata.org/) known from Python. Use this for higher performance. 
+    - [Query.jl](https://github.com/queryverse/Query.jl) Apply common data-analysis workflows like split-apply-combine, filter etc to almost any iterable Julia datastructure. Useful when working with data that's available in various different formats.  
+
+- Data input/output and storage: 
+    - [HDF5.jl](https://github.com/JuliaIO/HDF5.jl): For arbitrary data that should have a hierarchical organization in a single file. 
+    - [Arrow.jl](https://arrow.apache.org/julia/stable/): For arbitrary data that's organized in a tabular fashion.
+    - [CSV.jl](https://github.com/JuliaData/CSV.jl): For tabular, human readable data output. 
+
+- Visualization: 
+    - [Plots.jl](https://github.com/JuliaPlots/Plots.jl): Julia Frontend with a homogeneous interface for various well known plotting packages like [matplotlib](https://matplotlib.org/), [plotly](https://plotly.com/) or [GR](https://gr-framework.org/#). 
+    - [Makie.jl](https://github.com/MakieOrg/Makie.jl): Feature-complete plotting package in pure Julia. 
+    - [AlgebraOfGraphics.jl](https://github.com/MakieOrg/AlgebraOfGraphics.jl) Provides support for the grammar-of-graphics paradigm based on Makie.jl .
+    - [VegaLite.jl](https://github.com/queryverse/VegaLite.jl): Provides access to [VegaLite](https://vega.github.io/vega-lite/) from Julia, particularly useful for making interactive graphics. 
+
+- Machine Learning/Deep Learning: 
+    - [SciKitLearn.jl](https://github.com/cstjean/ScikitLearn.jl) for having access to the python based [scikit-learn](https://scikit-learn.org/) library in Julia. 
+    - [Flux.jl](https://github.com/FluxML/Flux.jl) for deep learning applications. 
+
+Have a look at [the JuliaHub package database](https://juliahub.com/ui/Packages) to find other packages for your use case. 
```

Comment:
The link is correct if I put the address in a browser, but if I click on this link I get this error:

![image](https://github.com/ssciwr/guidelines/assets/12845624/83fc6fb4-8e3f-4946-b809-1568e4d6fc00)

Just a note to check this link works after merging the changes

Reply:
hm, that's strange. tried it from code editor, chrome and firefox and it works...